variation-commons-beacon/src/main/java/uk/ac/ebi/eva/commons/beacon/models/Chromosome.java

```diff
@@ -0,0 +1,76 @@
+/*
+ * European Variation Archive (EVA) - Open-access database of all types of genetic
```

Comment:
Could the references to the EVA please be removed from the copyright headers? This library is supposed to be usable by other projects.

Reply:
I have removed them :)